--- a/openair2/NETWORK_DRIVER/MESH/device.c
+++ b/openair2/NETWORK_DRIVER/MESH/device.c
@@ -324,7 +324,7 @@ static const struct net_device_ops nasmesh_netdev_ops = {
   .ndo_set_mac_address  = NULL,
   .ndo_set_config     = nas_set_config,
   .ndo_do_ioctl       = nas_CTL_ioctl,
-#if  (defined RHEL_RELEASE_CODE && RHEL_RELEASE_CODE>=1797)
+#if  (defined RHEL_RELEASE_CODE && RHEL_RELEASE_CODE>=1797 && RHEL_RELEASE_CODE<RHEL_RELEASE_VERSION(8,0))
   .extended.ndo_change_mtu   = nas_change_mtu,
 #else
   .ndo_change_mtu   = nas_change_mtu,
