diff --git a/cmake_targets/build_oai b/cmake_targets/build_oai
index 6f4d560..fd727bc 100755
--- a/cmake_targets/build_oai
+++ b/cmake_targets/build_oai
@@ -633,14 +633,14 @@ function main() {
         $build_dir coding \
         libcoding.so $dbin/libcoding.so
 
-      compilations \
-        $build_dir nasmesh \
-        CMakeFiles/nasmesh/nasmesh.ko $dbin/nasmesh.ko
-
-      compilations \
-        $build_dir rb_tool \
-        rb_tool $dbin/rb_tool
-      cp $OPENAIR_DIR/cmake_targets/tools/init_nas_nos1 $dbin
+      #compilations \
+      #  $build_dir nasmesh \
+      #  CMakeFiles/nasmesh/nasmesh.ko $dbin/nasmesh.ko
+
+      #compilations \
+      #  $build_dir rb_tool \
+      #  rb_tool $dbin/rb_tool
+      #cp $OPENAIR_DIR/cmake_targets/tools/init_nas_nos1 $dbin
     fi
 
   fi
