--- a/libfdcore/sctp.c	2019-06-25 05:54:04.421402690 -0400
+++ b/libfdcore/sctp.c	2019-06-25 05:55:18.385811644 -0400
@@ -48,7 +48,7 @@
 /* #define OLD_SCTP_SOCKET_API */
 
 /* Automatically fallback to old API if some of the new symbols are not defined */
-#if (!defined(SCTP_CONNECTX_4_ARGS) || (!defined(SCTP_RECVRCVINFO)) || (!defined(SCTP_SNDINFO))) 
+#if (!defined(SCTP_CONNECTX_4_ARGS) || (!defined(SCTP_NOTIFICATIONS_STOPPED_EVENT)) || (!defined(SCTP_SNDINFO))) 
 # define OLD_SCTP_SOCKET_API
 #endif
 
